feature-libs/product/configurators/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { Configurator } from '../../../core/model/configurator.model';
+import { FormControl } from '@angular/forms';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-configurator-attribute-product-card',
+  templateUrl: './configurator-attribute-product-card.component.html',
+})
+export class ConfiguratorAttributeProductCardComponent
+  implements OnDestroy, OnInit {
+  quantity = new FormControl(1);
+  private subs: Subscription;
```

Comment:
why plural if you only have 1 sub? check the codebase and look how multiple subs are handled (adding subs to array-ish entity, that contains all of them and can unsub all at once)

Reply:
changed to single sub